package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "eslint ."
   },
   "lint-staged": {
-    "*.{js,jsx}": ["eslint --fix", "prettier --parser flow --write", "git add"],
+    "*.{js,jsx}": ["eslint --fix", "prettier --write", "git add"],
```

Comment:
Removing `--parser flow` allowed Prettier to parse the export-default-from syntax too.